draft-irtf-cfrg-vdaf.md

```diff
@@ -4983,7 +5036,7 @@ functions `extend()`, `convert()`, and `decode_public_share()` defined in
 def eval(
```

Comment:
Similarly, the preamble before this shouldn't mention `decode_public_share()` anymore